plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntitySheared.java

```diff
@@ -0,0 +1,88 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.nms.NMSHandler;
+import com.denizenscript.denizen.nms.NMSVersion;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.utilities.BukkitImplDeprecations;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import org.bukkit.entity.*;
+
+public class EntitySheared extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name sheared
+    // @input ElementTag(Boolean)
+    // @description
+    // Controls whether a sheep is sheared, a bogged is harvested or a snow golem is derped, ie not wearing a pumpkin.
+    // To include drops or harvesting mushroom cows consider <@link mechanism EntityTag.shear>.
```

Comment:
Nitpick, but
A. `bogged is harvested or a snow golem` missing `,`
B. `ie not wearing a pumpkin` probably in parenthesis?
C. `To include drops or harvesting mushroom cows consider` - `for harvesting`.